src/components/layout.tsx

```diff
@@ -51,6 +51,35 @@ const Layout = ({ children, title, description, img }: Props) => {
       doc.body.setAttribute('style', `--magic-hero-number: ${356 - offset}px`);
     }
     window.requestAnimationFrame(magicHeroNumber);
+
+    if (doc.body.dataset.browser !== 'legacy') {
+      doc.body.dataset.browser = 'legacy';
+
+      const css = doc.createElement('style');
+      css.type = 'text/css';
+      const styles = `
+        .side-nav__title {
```

Comment:
Since we're already applying a data attribute, can we move this logic into the main CSS file and just change this selector to `[data-browser=legacy] .side-nav__title`?

Reply:
Fantastic suggestion! The code is so much better organised and feels less hacky after doing this. ❤️